integration/spark-common/src/main/scala/org/apache/carbondata/spark/util/CommonUtil.scala

```diff
@@ -369,6 +369,14 @@ object CommonUtil {
     }
   }
 
+  def validateListInfo(listInfo: List[List[String]]): Unit = {
```

Comment:
two more things:
1. this function is only used once, so I think there is no need to place it in `CommonUtil`.
2. is `distinct` case-sensitive or not?